.github/workflows/test.yml

```diff
@@ -0,0 +1,40 @@
+name: Test
```

Comment:
If this workflow only runs the unit test, could we rename the workflow to be "unit_test" instead to be more clear?

Reply:
Ah that makes sense - ignore this comment then, happy to approve :)